ambari-server/src/main/java/org/apache/ambari/server/state/stack/UpgradePack.java

```diff
@@ -156,76 +152,18 @@ public UpgradeType getType() {
     if (prerequisiteChecks == null) {
       return new ArrayList<>();
     }
-    return new ArrayList<>(prerequisiteChecks.checks);
-  }
 
-  /**
-   *
-   * @return the prerequisite check configuration
-   */
-  public PrerequisiteCheckConfig getPrerequisiteCheckConfig() {
-    if (prerequisiteChecks == null) {
-      return new PrerequisiteCheckConfig();
-    }
-    return prerequisiteChecks.configuration;
+    return prerequisiteChecks.stream().map(c -> c.className).collect(Collectors.toList());
```

Comment:
Just double-checking - is classname here the FQDN or the simple name? It needs to be the

Reply:
FQDN